infra/debian/compiler/docs/one-profile.1

```diff
@@ -0,0 +1,37 @@
+.TH ONE-PROFILE "1" "August 2021" "one-profile version 1.16.1" "User Commands"
+.SH NAME
+one-profile \- profile backend model file
+.SH DESCRIPTION
+usage: one\-profile [\-h] [\-v] [\-C CONFIG] [\-b BACKEND] [\-\-] [COMMANDS FOR BACKEND]
+.PP
+\fBone\-profile\fR is a command line tool for profiling backend model.
+.SH OPTIONS
+.TP
+\fB\-h\fR, \fB\-\-help\fR
+show this help message and exit
+.TP
+\fB\-v\fR, \fB\-\-version\fR
+show program's version number and exit
+.TP
+\fB\-C\fR CONFIG, \fB\-\-config\fR CONFIG
+run with configuation file
+.TP
+\fB\-b\fR BACKEND, \fB\-\-backend\fR BACKEND
+backend name to use (There is no available backend
+drivers)
```

Comment:
```suggestion
backend name to use
```

Reply:
I agree. one-codegen man page has the same code. I will update that too.